static/app/views/settings/organizationMembers/organizationMemberRow.tsx

```diff
@@ -237,10 +237,17 @@ export default class OrganizationMemberRow extends PureComponent<Props, State> {
 
 const StyledPanelItem = styled(PanelItem)`
   display: grid;
-  grid-template-columns: minmax(150px, 2fr) minmax(90px, 1fr) minmax(120px, 1fr) 90px;
+  grid-template-columns: minmax(150px, 4fr) minmax(90px, 2fr) minmax(120px, 2fr) minmax(
+      100px,
+      1fr
+    );
   gap: ${space(2)};
   align-items: center;
 `;
+// Force action button at the end to align to right
+const RightColumn = styled('div')`
+  margin-left: auto;
```

Comment:
IMO I would use this instead
```suggestion
  display: flex;
  justify-content: flex-end;
```

but that's totally personal opinion, especially since it's an extra line so 🤷

Reply:
Yea makes sense. I've added it in 2c5b3740f99dc65f1050b9b3477c5c8106d6c8ad